### PR TITLE
HRING-2965 [Demo] Spatial (Node.js)

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

**Comment by Danielle9897** [comment]
For issue:
https://issues.hibernatingrhinos.com/issue/HRINT-2965/Demo-Spatial-Nodejs

File: DemoServer/AdditionalLanguages/nodejs/demo/spatial/spatialQuery/index.js [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
 
+    //region Step_2
     const employeesWithinCircle = await session.query(Employee)
         .spatial(
+    //endregion
+            //region Step_3
             new PointField('Address.Location.Latitude', 'Address.Location.Longitude'),
+            //endregion
+            //region Step_4
             spatialCriteria => spatialCriteria.relatesToShape(wktCircle, 'Within', 'Miles', 0)
```
Comment (Danielle9897) [inline DemoServer/AdditionalLanguages/nodejs/demo/spatial/spatialQuery/index.js:28]: @ml054 
The C# client has a default value for distErrorPercent `double distErrorPercent = 0.025`
 but here (and in Java client as well)
 we pass `0` for that param as no default value is defined in method signature.
 
 Can that be added ?

Comment (ml054) [reply DemoServer/AdditionalLanguages/nodejs/demo/spatial/spatialQuery/index.js:28]: just remove this param and match c# version

Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/nodejs/demo/spatial/spatialQuery/index.js:28]: can't just remove the `0` from the Java demo
no matching overload method for that

**Pull request merged**